praetorian_cli/sdk/entities/jobs.py

```
@@ -8,7 +8,7 @@ class Jobs:
     def __init__(self, api):
         self.api = api
 
-    def add(self, target_key, capabilities=[], config=None):
+    def add(self, target_key, capabilities=[], config=None, credentials=[]):
```

Ruff (0.14.3)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)